Formula/gstreamer.rb

```diff
@@ -51,6 +51,8 @@ def install
       system "ninja", "-v"
       system "ninja", "install", "-v"
     end
+
+    bin.env_script_all_files libexec/"bin", GST_PLUGIN_SYSTEM_PATH: HOMEBREW_PREFIX/"lib/gstreamer-1.0"
```

Comment:
Just a temporary workaround to unblock testing.

Not ideal as we lose default plugin lookup behavior, which searched the user's XDG data directory and then the system directory (e.g. `PLUGINDIR`).

However, the problem is `gstreamer` code is dynamically looking up location of shared library rather than using our `PLUGINDIR` inreplace.

---

Note that `GST_PLUGIN_PATH` has higher priority than `GST_PLUGIN_SYSTEM_PATH`.

Reply:
We already do an `inreplace` above. I guess another one might not hurt, if this is feasible to fix with an `inreplace`.

Reply:
An `inreplace` would need to be done on source code (https://gitlab.freedesktop.org/gstreamer/gstreamer/-/commit/747a82006cfc0223a53d44edfa56362b6177f0c9).

For example, can change function that gets library location in `gst/gstpluginloader.c` by any of following:
- Remove `dir = g_path_get_dirname (info.dli_fname);` 
- Change `dladdr (&gst_init, &info)` to some false condition
- Make directive `#elif defined(__APPLE__) && defined(HAVE_DLADDR)` be false.

Or can modify location of function calls like:
- Change `if (relocated_libgstreamer)` to `if (NULL)`
- Change `priv_gst_get_relocated_libgstreamer ();` to `NULL;`

Reply:
I see. Probably not worth it then. We should consider opening an issue upstream asking for a way to configure `GST_PLUGIN_SYSTEM_PATH` at build time.